src/bootlint.js

```diff
@@ -964,6 +981,62 @@ var LocationIndex = _location.LocationIndex;
             }
         });
     });
+    addLinter("W015", function lintNewBootstrap($, reporter) {
+        var BOOTSTRAP_V4_ERROR = "Detected what appears to be Bootstrap v4. Bootlint currently supports Bootstrap version v3.";
+        var PLUGINS = [
+            'affix',
+            'alert',
+            'button',
+            'carousel',
+            'collapse',
+            'dropdown',
+            'modal',
+            'popover',
+            'scrollspy',
+            'tab',
+            'tooltip'
+        ];
+        var theWindow = getBrowserWindowObject();
+
+        var globaljQuery = theWindow && (theWindow.$ || theWindow.jQuery);
+        /* @covignore */
+        if (globaljQuery) {
+            var versions = getSemverFromPlugin(PLUGINS, globaljQuery);
+            if (versions.length) {
+                var minVersion = versions[0];
+                if (semver.gte(minVersion, BOOTSTRAP_VERSION_4, true) &&
+                    semver.lt(minVersion, BOOTSTRAP_VERSION_5, true)) {
+                    reporter(BOOTSTRAP_V4_ERROR);
+                    return;
+                }
+            }
+        }
+        // check for Bootstrap <link>s and <script>s
+        var bootstraps = $([
```

Comment:
Create a global constant whose value is the string resulting from this list join. Then use `bootstraps = $(THE_CONSTANT)` here.